scripts/release/workflow/start.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+# Set changeset status location
+# This is needed because `changeset status --output` only works with relative routes
+CHANGESETS_STATUS_JSON=$(node -p -e "require('path').relative(__dirname, '$RUNNER_TEMP/status.json')")
```

Comment:
This shell script has a bit of an identity crisis with JS. :sweat_smile: 

Also it's preferable to use quotes around honestly everything that has a $ on it.

```suggestion
CHANGESETS_STATUS_JSON="$(realpath --relative-to=. "$RUNNER_TEMP/status.json")"
```

Reply:
> This shell script has a bit of an identity crisis with JS. 😅
Oh, I know, but making a JS script will also have an identity crisis with shell haha
I solved it with your other suggestion, thanks!

Same answer as [here](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3915#discussion_r1072439360)